client/web-sveltekit/src/lib/search/resultsIndicator/SuggestedAction.svelte

```diff
@@ -9,85 +9,66 @@
     export let severity: string
     export let state: 'error' | 'complete' | 'loading'
 
+    const SEE_MORE = 'See more details'
     const CENTER_DOT = '\u00B7' // AKA 'interpunct'
 
-    $: sortedItems = sortBySeverity(progress.skipped)
-    $: suggestedItems = sortedItems.filter((skipped): skipped is Required<Skipped> => !!skipped.suggested)
+    interface ItemsBySeverity {
+        items: Skipped[]
+        mostSevere: Skipped | null
+    }
+
+    function filterAndSortItems(items: Skipped[]): ItemsBySeverity {
+        if (items.length === 0) {
+            return { items, mostSevere: null }
+        }
+
+        const filteredItems = items.filter((item: Skipped) => {
+            const isNotFork = item.reason !== 'repository-fork'
+            const isNotArchive = item.reason !== 'excluded-archive'
+            return isNotFork && isNotArchive
```

Comment:
maybe easier to read?

```suggestion
            return !['repository-fork', 'excluded-archive'].includes(item.reason)
```